main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma")
```

Comment:
There's a typo in the genre name 'Dramma'. It should be 'Drama'. Although you update it later, it's better to use 'Drama' from the start to avoid confusion.